### PR TITLE
Documentation updates and fixes for overriding various asset types

### PR DESCRIPTION
**Pull request by baz8080**
### Changes

* Updates example code, README, and CHANGELOG
* Default to `UnityTheme` on Android for a better OOB
* Resets gradle.properties to the AS default
* Adds gradle script to invoke zip when building to avoid an AAPT error about duplicate CSS files when using the standard build system on Unity 5.x

### Reviewers
@brendan-fahy @baz8080 @schlan @ndobri @oarrabi @tecknut @StevenDiviney @RonanMcH

### FYI
@KevinLambe @mathewcropper 

### Risks
- Medium

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
 
+This is a Unity plugin that wraps the iOS and Android Zendesk SDKs.
 
 ## Requirements
 
 - OSX or Linux operating system (The plugin is not supported on Windows)
 - Unity 5.0+
-- Xcode 7.0+ (if building for iOS)
-- Android SDK with the latest support repo and libraries installed. (Required to build the plugin, even if you are just using iOS)
-- Note that you must have the "Android Support Library" installed. You can download this using the Android SDK Manager.
```
Comment (brendan-fahy) [inline README.md:15]: I can't remember why this was required, but I remember a couple of months ago, I couldn't build until I did it. Can you remember what the deal was here, and why it's no longer needed?

File: android-plugin/build.gradle [hunk 2]
```diff
@@ -49,12 +67,48 @@ task packagePlugin <<  {
...
+ *
+ * This will not work on Windows. If you are trying to build on Windows, then do not call this
+ * task from packagePlugin, and instead delete 'assets/help_center_article_style.css' from
+ * $rootProject.buildDir/unity-plugin/Plugins/Android/sdk-a.b.c.d.aar
+ */
+task stripCssFromAndroidSdk(type:Exec) {
+
+    /*
+        ignoreExitValue can be uncommented to get detailed error messaging if the zip task fails.
+        If the code is 12 then it is likely that zip cannot find the file, or the file has alread
```
Comment (brendan-fahy) [inline android-plugin/build.gradle]: Abrupt finish - the file has already something something?

Comment (baz8080) [reply android-plugin/build.gradle]: thanks, will fix. `already been processed`

File: README.md (hunk 1 above)
Comment (baz8080) [reply README.md:15]: Yes, now the required aars (not the legacy one pointed at above) are included in `android-plugin/plugin-dependencies`

**Review by brendan-fahy — APPROVED** [review]
:+1:
Nice, Mrs Pancakes. _Nice_.

**Pull request merged**